velodyne_driver/include/velodyne_driver/driver.hpp

```diff
@@ -71,6 +71,7 @@ class VelodyneDriver final : public rclcpp::Node
     int cut_angle;                   // cutting angle in 1/100°
     double time_offset;              // time in seconds added to each velodyne time stamp
     bool enabled;                    // polling is enabled
+    bool timestamp_first_packet;
```

Comment:
Nit: can you please add a comment to this line, to be consistent with the rest of the structure?

Reply:
@clalancette add comment at https://github.com/ros-drivers/velodyne/pull/436/commits/c78129f23ed448b8a0a2fa6dc1514b577aeed331